### ISSUE TITLE
Game crashes when changing a "yes/no" option value

### ISSUE THREAD

**Issue by fedorov-ao** [yairm210/Unciv#2366]
**Bug description**

The game crashes when a user changes a value of "yes/no" option in options menu. This affects all "yes/no" options like "Show worked tiles", "Show tutorials", but not "drop-down list" options like "Language" or "Resolution". 

**Versions affected**

At least 32-bit Linux versions of [3.7.1](https://github.com/yairm210/Unciv/releases/tag/3.7.1) and [3.7.1-linux32test](https://github.com/yairm210/Unciv/releases/tag/3.7.1-linux32test) . Last working release is [3.7.0](https://github.com/yairm210/Unciv/releases/tag/3.7.0) . Please note that 32-bit Linux versions of 3.7.0 and 3.7.1 releases does not work "out of the box", more info [here](https://github.com/yairm210/Unciv/issues/2361) .

**Steps to Reproduce**

1. Launch game
2. Open main menu
3. Click "Options"
4. Click on any of the "yes/no" options
5. The game will crash

Example of the shell output:
```
$ java -jar Unciv.jar 
Packing textures - 71ms
Loading game basics - 1171ms
Loading translation file for English - 374ms
Loading percent complete of languages - 32ms
Loading characters for font - 28ms
Putting glyphs - 8631ms
RNG seeded with 1586367252007
Natural Wonders for this game: [Great Barrier Reef, Old Faithful]
No suitable location for Great Barrier Reef
Exception in thread "LWJGL Application" java.lang.RuntimeException: Actor: AutoScrollPane
|  Table
|  |  Label: Display options
|  |  Label: Show worked tiles
|  |  TextButton
|  |  |  Label: No
|  |  La
        at com.badlogic.gdx.scenes.scene2d.Actor.notify(Actor.java:202)
        at com.badlogic.gdx.scenes.scene2d.Actor.fire(Actor.java:158)
        at com.badlogic.gdx.scenes.scene2d.Stage.act(Stage.java:209)
        at com.badlogic.gdx.scenes.scene2d.Stage.act(Stage.java:186)
        at com.unciv.ui.utils.CameraStageBaseScreen.render(CameraStageBaseScreen.kt:49)
        at com.unciv.ui.worldscreen.WorldScreen.render(WorldScreen.kt:539)
        at com.badlogic.gdx.Game.render(Game.java:46)
        at com.badlogic.gdx.backends.lwjgl.LwjglApplication.mainLoop(LwjglApplication.java:233)
        at com.badlogic.gdx.backends.lwjgl.LwjglApplication$1.run(LwjglApplication.java:128)
Caused by: java.lang.IllegalStateException: stage must not be null
        at com.unciv.ui.utils.AutoScrollPane$1.exit(AutoScrollPane.kt:47)
        at com.badlogic.gdx.scenes.scene2d.InputListener.handle(InputListener.java:72)
        at com.badlogic.gdx.scenes.scene2d.Actor.notify(Actor.java:188)
        ... 8 more
```

**Comment by JackRainy** [comment]
The bug has been introduced only in this particular build 3.7.1-linux32test (3.7.1 should work).
The fix is coming with PR #2369

**Comment by yairm210** [comment]
Solved, thanks to all involved =)
